mousetrap.js

```diff
@@ -1018,4 +1023,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
I've updated my pull request.
Replaced typeof document with typeof window check, because we still can't explicitly pass document as an argument if environment is not browser (i.e. window is undefined), that's going to cause an exception.